overlord/hookstate/ctlcmd/ctlcmd.go

```diff
@@ -91,19 +91,44 @@ func addCommand(name, shortHelp, longHelp string, generator func() command) {
 	}
 }
 
+type ForbiddenCommand struct {
+	Uid     uint32
+	Command string
+}
+
+func (f ForbiddenCommand) Error() string {
+	return fmt.Sprintf("cannot use %q with uid %d, try with sudo", f.Command, f.Uid)
+}
+
+func (f *ForbiddenCommand) Execute(args []string) error {
+	return f
```

Comment:
That seems unnecessarily magical. We can do this here instead:
```
return fmt.Errorf("cannot use %q with uid %d, try with sudo", f.Command, f.Uid)
```
and drop the error method.

Reply:
The motivation for this was to have a distinct error type, because we need to differentiate 'forbidden' error from anything else in `api.go`. How about a new error type - ForbiddenCommandError, so it becomes less magical and more explicit?

Reply:
Add `ForbiddenCommandError`, what do you think?